cpu/stm32_common/periph/flashpage.c

```diff
@@ -7,7 +7,7 @@
  */
 
 /**
- * @ingroup     cpu_stm32f0
+ * @ingroup     cpu_stm32_common
```

Comment:
Agree it should run fine on f3, but on f2 and f4, the sectors don't all have the same size.

It's up to you, but I think a comment telling it has been tested only on f0 and f1 would be enough then.